test/unit/theory/theory_int_opt_white.h

```diff
@@ -0,0 +1,78 @@
+#include <iostream>
+#include <cxxtest/TestSuite.h>
+#include "api/cvc4cpp.h"
+#include "smt/optimization_solver.h"
+#include "expr/expr_manager.h"
+#include "expr/node.h"
+#include "expr/node_builder.h"
+#include "expr/node_manager.h"
+#include "expr/node_value.h"
+#include "smt/smt_engine.h"
+#include "smt/smt_engine_scope.h"
+
+using namespace CVC4;
+using namespace smt;
+using namespace api;
+
+class omt : public CxxTest::TestSuite {
+  NodeManager *d_nm;
+  SmtEngine *d_smt;
+  SmtScope *d_scope;
+  api::Solver* d_slv;
+  OptimizationSolver* d_optslv;
+ public:
+  void setUp() override
+  {
+    //d_optslv(nullptr)
+    //d_optslv.reset(new OptimizationSolver(this));
+
+    d_slv = new api::Solver();
+    d_smt = d_slv->getSmtEngine();
+    //d_nm = d_slv->getNodeManager();
+    d_nm = NodeManager::currentNM();
+    d_scope = new SmtScope(d_smt);
+    d_optslv = new OptimizationSolver(d_smt);
+
+    Options opts;
+    d_slv->setOption("produce-assertions", "true");
+    d_slv->setOption("incremental", "true");
+  }
+  void tearDown() override
+  {
+    //delete d_nm;
+    delete d_optslv;
+    delete d_scope;
+    delete d_slv;
+  }
+  void testOmt()
+  {
+    Term ub = d_slv->mkReal(100);
```

Comment:
Please add documentation: what formula are you creating? what is the expected solution?
It is best to add some comments before each block of code to explain what it does.